src/edu/ucsb/nceas/metacat/restservice/D1ResourceHandler.java

```diff
@@ -44,6 +45,7 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.dataone.client.v1.itk.D1Client;
```

Comment:
```suggestion
import org.dataone.client.v2.itk.D1Client;
```
Should we be using the V2 client regularly? Maybe this is an auto-import typo?

Reply:
Yes, we should use the v2 client.